engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java

```diff
@@ -646,62 +677,66 @@ private void removeRigidBody(BulletRigidBody body) {
      * <br><br>
      * TODO: Flyweight this (take scale as parameter)
```

Comment:
Is this TODO still valid? If so maybe could it be expanded on a bit to make it clear what it means?